src/renderer/components/swap/Swap.utils.ts

```diff
@@ -156,8 +156,8 @@ export const getSwapData = ({
  */
 export const getSwapLimit = (swapResultAmountMax1e8: BaseAmount, slipTolerance: SlipTolerance): BaseAmount => {
   const swapLimit: BaseAmount = swapResultAmountMax1e8.times(1.0 - slipTolerance * 0.01)
-  const swapLimitWithIdentifier: number = +swapLimit.amount().toString().slice(0, -3).concat(ASGARDEX_SWAP_IDENTIFIER)
-  return baseAmount(bn(swapLimitWithIdentifier))
+  const swapLimitWithIdentifier = +swapLimit.amount().toString().slice(0, -3).concat(ASGARDEX_SWAP_IDENTIFIER) - 1000
+  return baseAmount(bn(swapLimitWithIdentifier <= 999 ? 999 : swapLimitWithIdentifier))
```

Comment:
Use `ASGARDEX_SWAP_IDENTIFIER` instead of `999`. Change it before to be a number in `renderer/const.ts`